app/controllers/volunteer_ops_controller.rb

```diff
@@ -97,11 +101,15 @@ def current_user_has_organisation?
   end
 
   def set_organisation
-    @organisation = Organisation.friendly.find(params[:organisation_id]) rescue nil
+    @organisation = Organisation.friendly.find(params[:organisation_id])
+    rescue ActiveRecord::RecordNotFound
+      @organisation = nil
   end
 
   def set_volunteer_op
     @volunteer_op = VolunteerOp.find(params[:id])
+    rescue ActiveRecord::RecordNotFound
+      @volunteer_op = nil
```

Comment:
modified "set_volunteer_op" method to provide needed functionality to pass new cucumber test in "custom_error_pages.feature"